packages/node_modules/@ciscospark/widget-message/src/container.js

```diff
@@ -722,14 +720,12 @@ export default connect(
     loadMissingActivities,
     loadPreviousMessages,
     removeFlagFromServer,
-    receiveMercuryActivity,
-    receiveMercuryComment,
     removeInflightActivity,
     retryFailedActivity,
     setScrollPosition,
     setTyping,
     showScrollToBottomButton,
     updateHasNewMessage,
-    updateMercuryState
+    updateWidgetState
   }, dispatch)
-)(MessageWidget);
+)(ConnectedMessageWidget);
```

Comment:
Is this the right order of applying `connect()`? I can't tell if this is right or the order should be reversed like:
``` js
const MessageWidget = connect(getMessageWidgetProps, blah blah)(MessageWidget);
export default wrapConversationMercury(MessageWidget);

Reply:
I would think we would need to connect the wrapped component so it gets all the redux stores it needs.